### DISCUSSION TITLE
Extract the data using button on a website?

### DISCUSSION THREAD

**Discussion by cmarabate** [LukeCroteau/RaidExtractor#45] [Q&A] [answered]
I'm just curious if anyone knows a way to possibly add a button to a website that could somehow extract the JSON file from your Raid account while logged-in to Plarium Play just like the the exe file does? It would be nice if it automatically stored all the JSON to a localStorage key that I could choose but I could always make that happen if I was able to accomplish the first part.

If anyone knows a way to pull this off I would be very interested. Until then, I am happy with having a file upload field for visitors to upload their JSON file from the RaidExtractor that stores the entire contents of it to a localStorage key.

Thanks,
Chris

**Comment by sphyrnus** [comment, marked as answer]
A web browser cannot directly access contents of the file system (security), upload of files must be initiated by the client.

If you have some kind of an application installed on the host, it would be possible to send a command to that application to then initiate an upload in turn, however.

**Reply by cmarabate** [reply]
That makes sense, thanks!
